### PR TITLE
feat: fix all migrations

### PR DESCRIPTION
**Pull request by danielgent**
This PR
* reverts the logs table migration to how it was ran on production
* copies in the sessions and passwords migrations from the feature branch
* creates a new migration to add the new fields to the logs table :facepalm: 

*I THINK THAT IS ALL AND THIS WILL BE GOOD FOR PROD* :grimacing:

### CODE REVIEW

File: src/migrate/migrations/2023-10-26T08:14:09_log_table.ts [hunk 1]
```diff
@@ -5,11 +5,8 @@ export async function up(db: Kysely<Database>): Promise<void> {
   await db.schema
     .createTable("logs")
     .addColumn("id", "varchar(255)", (col) => col.notNull().primaryKey())
-    .addColumn("tenant_id", "varchar(255)", (col) =>
-      col.references("tenants.id").onDelete("cascade").notNull(),
-    )
-    .addColumn("user_id", "varchar(255)", (col) =>
-      col.references("users.id").onDelete("cascade").notNull(),
+    .addColumn("client_id", "varchar(255)", (col) =>
+      col.references("clients.id").onDelete("cascade").notNull(),
```
Comment (danielgent) [inline src/migrate/migrations/2023-10-26T08:14:09_log_table.ts:9]: This is how it was ran on production :facepalm: 

https://github.com/sesamyab/auth/commit/e88928c6a3ac3b4960977185db01bb0155c6dad7

image

File: src/migrate/migrations/2023-11-02T23:18:12_sessions.ts
Comment (danielgent) [inline src/migrate/migrations/2023-11-02T23:18:12_sessions.ts:1]: nothing fancy in git with cherry-picking commits! I just copy-pasted these in

File: src/migrate/migrations/2023-11-08T17:12:09_logs-table-new-fields.ts
Comment (danielgent) [inline src/migrate/migrations/2023-11-08T17:12:09_logs-table-new-fields.ts:1]: this now correctly alters the table as per the diff screenshot on the first comment

File: src/migrate/migrations/2023-11-08T17:12:09_logs-table-new-fields.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import { Kysely } from "kysely";
+import { Database } from "../../types";
+
+export async function up(db: Kysely<Database>): Promise<void> {
+  await db.schema
+    .alterTable("logs")
+    .dropColumn("client_id")
+    .addColumn("tenant_id", "varchar(255)", (col) =>
```
Comment (markusahlstrand) [inline src/migrate/migrations/2023-11-08T17:12:09_logs-table-new-fields.ts:8]: I guess the columns already exists in dev? Maybe we could comment it out before running the migrations in dev?

**Review by markusahlstrand — APPROVED** [review]
Looks good.

**Pull request merged**